src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/LicenceController.kt

```diff
@@ -93,8 +95,9 @@ class LicenceController(
   fun createLicence(
     @RequestBody @Valid
     request: CreateLicenceRequest,
-  ): LicenceSummary {
-    return licenceCreationService.createLicence(request.nomsId)
+  ): LicenceSummary = when (request.type) {
+    CRD -> licenceCreationService.createLicence(request.nomsId)
+    HARD_STOP -> licenceCreationService.createHardStopLicence(request.nomsId)
   }
```

Comment:
used existing endpoint and the new type field determines which type of licence to create